File: Part 1 - Data Preprocessing/data_preprocessing_template.R
# Plantilla Preprocesado de datos

#importando dataset

dataset = read.csv('Data.csv')
# dataset = dataset[,2:3]
#Dividir data set entre conjunto de entranamiento y testing
library(caTools)

set.seed(123) #Seleccionando semilla
#sample.split(Datos a separar, porcentaje para entrenar)
split = sample.split(dataset$Purchased,SplitRatio = 0.8)
#Separando datos de entrenamiento y testing
training_set = subset(dataset,split==TRUE)
testing_set = subset(dataset,split==FALSE)

#Escalado de valores
#[filas,columnas]
# training_set[,2:3] = scale(training_set[,2:3])#[Todas las filas, de 2 a 3]
# testing_set[,2:3] = scale(testing_set[,2:3])

File: Part 1 - Data Preprocessing/data_preprocessing_template.py
# -*- coding: utf-8 -*-
"""
Created on Fri Mar 24 17:15:54 2023

@author: josue
"""

# Plantilla de preprocesado de datos

import numpy as np
import matplotlib.pyplot as plt
import pandas as pd

from sklearn.impute import SimpleImputer
from sklearn import preprocessing
from sklearn.preprocessing import OneHotEncoder, StandardScaler
from sklearn.compose import ColumnTransformer
from sklearn.model_selection import train_test_split

dataset = pd.read_csv('Data.csv')

X = dataset.iloc[:, :-1].values
y = dataset.iloc[:, 3].values  # vector

#Dividir data set entre conjunto de entranamiento y testing
X_train,X_test,y_train,y_test=train_test_split(X,y,test_size = 0.2, random_state=0)

#Escalado de variables
"""sc_X = StandardScaler()
X_train = sc_X.fit_transform(X_train)
X_test = sc_X.transform(X_test)"""



















